src/solver/modeler/main.cpp

```diff
@@ -105,6 +105,22 @@ int main(int argc, const char** argv)
         const auto system = LoadFiles::loadSystem(studyPath, libraries);
         logs.info() << "System loaded";
         SystemLinearProblem system_linear_problem(system);
+
+        auto outputPath = studyPath / "output";
+        if (!parameters.noOutput)
+        {
+            logs.info() << "Output folder : " << outputPath;
+            if (!is_directory(outputPath))
+            {
+                if (!create_directory(outputPath))
+                {
+                    logs.error() << "Failed to create output directory. Exiting simulation.";
+                    throw std::runtime_error(
+                      "Failed to create output directory. Exiting simulation.");
```

Comment:
Currently, this exception isn't caught. I suggest instead
```suggestion
                    return EXIT_FAILURE;
```